koku/api/provider/models.py

```diff
@@ -144,6 +144,28 @@ class Meta:
     created_timestamp = models.DateTimeField(auto_now_add=True, blank=True, null=True)
 
 
+class Sources(models.Model):
+    """Platform-Sources table.
+
+    Used for managing Platform-Sources.
+    """
+
+    class Meta:
+        """Meta for Sources."""
+
+        db_table = 'api_sources'
+
+    source_id = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=256, null=True)
+    source_type = models.CharField(max_length=50, null=False)
+    authentication = models.CharField(max_length=128, null=False)
+    billing_source = models.CharField(max_length=128, null=True)
+    koku_uuid = models.CharField(max_length=512, null=True)
```

Comment:
I would consider making this a real foreign key to our provider table. That would also make it clearer that this is a provider uuid.

Reply:
Yes, using uuid for now since all provider sync communication is still HTTP based.  I agree to update this if we ever change that to be direct database updates.